src/translations/nb_NO.json

```diff
@@ -140,7 +140,7 @@
   "FOREVER_INTRO_READY_CTA": "Ja, gi meg et prisforslag.",
   "FOREVER_LANDINGPAGE_INPUT_TEXT": "Fyll inn koden",
   "FOREVER_LANDINGPAGE_BTN_LABEL": "Neste",
-  "FOREVER_LANDINGPAGE_INFO_TEXT": "Dette er en eksklusiv nettside som er tiltenkt folk som er invitert. Hvis du for øyeblikket ikke har fått en invitasjon enda, så frykt ikke. Du kan fortsatt melde deg inn på (web onboarding link til NO)",
+  "FOREVER_LANDINGPAGE_INFO_TEXT": "Dette er en eksklusiv nettside som er tiltenkt folk som er invitert. Hvis du for øyeblikket ikke har fått en invitasjon enda, så frykt ikke. Du kan fortsatt melde deg inn på (https:\/\/www.hedvig.com\/no\/new-member)",
```

Comment:
i understand this is from lokalise but shouldn't this be something like `[https:\/\/www.hedvig.com\/no\/new-member](https:\/\/www.hedvig.com\/no\/new-member)`?

Reply:
True, let's fix that